storage/container_storage_interface/persistent-storage-csi-cinder.adoc

```diff
@@ -0,0 +1,27 @@
+[id="persistent-storage-csi-cinder"]
+= OpenStack Cinder CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-cinder
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for OpenStack Cinder.
+
+:FeatureName: OpenStack Cinder CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
```

Comment:
I was under the impression Cinder CSI was fully supported in 4.7. @Fedosin and @jsafrane , do you confirm?

Reply:
Yeah, I'm not sure how it will land. @duanwei33, you [suggested](https://github.com/openshift/openshift-docs/pull/28015#issuecomment-755021455) adding to the Release Notes in TP but to keep an eye on it as we get closer. Until someone confirms it will be going GA, I'll leave it as TP.

Reply:
Yeah, it is fully supported